latest/index.bs

```diff
@@ -207,8 +207,52 @@ keys as specified below for discovering certain types of data, especially images
 
 Metadata about the multiple resolution representations of the image can be
 found under the "multiscales" key in the group-level metadata.
-The specification for the multiscale (i.e. "resolution") metadata is provided
-in [zarr-specs#50](https://github.com/zarr-developers/zarr-specs/issues/50).
+"multiscales" contains a list of dictionaries where each entry describes a multiscale image.
+
+"multiscales" MUST contain the field "datasets", which is a list of dictionaries describing
```

Comment:
It is a bit confusing to have `"multiscales" contains a list of dictionaries...` followed by `"multiscales" MUST contain the field "datasets"...`
I think the second sentence needs to say `Each "multiscales" dictionary MUST contain the field "datasets"...`

Reply:
Good point.